### DISCUSSION TITLE
Not getting the "push rejected" dialog over Wsl2

### DISCUSSION THREAD

**Discussion by RussKie** [gitextensions/gitextensions#10776] [Q&A]
I'm trying to push a branch after an amend which, naturally, is getting rejected. But where normally on Windows I'm getting the "push rejected" dialog, I'm not getting the same over the wsl2.
![image](https://user-images.githubusercontent.com/4403806/221479848-68822f2a-a709-44c6-bc19-49692c61d00a.png)

It looks like a bug - i.e., the app isn't getting notified of the err stream. But I wanted to check whether anyone else is observing the same.

**Comment by gerhardol** [comment]
Worked fine for me with pre WSL 1.0 (as Windows feature, not from Microsoft store). Got a replacement PC today (as the updates needed to install WSL 1.0 failed to be applied to my standard PC, it need to be reinstalled) and got this myself today with WSL 1.0.3.0.

**Reply by RussKie** [reply]
Interesting. I wonder now whether it's something wsl2-related...

**Comment by gerhardol** [comment]
In FormProcess.cs: FormStatusOutputLog OutputLog  is never updated, DataReceivedCore() is never called.
I do not see why.

**Comment by RussKie** [comment]
Yep, my observations too.
I spent about an hour trying to debug, but didn't get anywhere.
